### PR TITLE
Add handling for missing ACKs from the slave.

### LINKED ISSUE #16: I2C questions
Hi there.

Is the I2C implementation supposed to be working or is it still a bit too early?
My I2C circuit always seemed to reply with ack even when the power was off (using an ftdi calbe to talk with a seperate board) and then i started looking through the code a bit.
I noticed in the source things like ack's not getting handled.
Haven't looked at everything and have mostly toyed with the write function.

Like what you are doing so far and hope to replace my own hacky wrapper around libftd2xx-rs with this in the future.

Edit: Scratch the part about SDA not being driven high. That seems to have been my own fault.

### PR DESCRIPTION
**Pull request by newAM**
Closes #16.

Still needs more physical validation.

### CODE REVIEW

File: src/i2c.rs [hunk 1]
```diff
@@ -197,26 +322,100 @@ impl<'a, Device: FtdiCommon> embedded_hal::blocking::i2c::Write for I2c<'a, Devi
...
+                for _ in 0..self.start_stop_cmds {
+                    mpsse_cmd = mpsse_cmd
+                        .set_gpio_lower(inner.value | SCL | SDA, SCL | SDA | inner.direction)
+                }
+            }
+
+            mpsse_cmd = mpsse_cmd
+                // Idle
+                .set_gpio_lower(inner.value, inner.direction)
+                .send_immediate();
```
Comment (Blisto91) [inline src/i2c.rs]: You have to move this up into the last byte condition. Currently it fails when you write more than one byte using ``write_slow``.

Comment (newAM) [reply src/i2c.rs]: Good catch!  Made the change.

**Comment by Blisto91** [comment]
Tested to be working with my FT232H cable and teensy.

Maybe add a changelog entry for ``set_fast`` and the new slow default behavior.

**Comment by newAM** [comment]
Sorry for the slow response on my end, work kept me busy during the week!

> Tested to be working with my FT232H cable and teensy.

Thank you!  I really appreciate you testing this out!
Tested to be working on my FT232H + BME280 as well.

> Maybe add a changelog entry for set_fast and the new slow default behavior.

Done, good catch!

---

I also bumped the version to 0.7.0 in this PR, going to do a release right away.

**Pull request merged**